osquery/tables/system/CMakeLists.txt

```diff
@@ -167,6 +167,12 @@ function(generateOsqueryTablesSystemSystemtable)
       darwin/xprotect.cpp
     )
 
+    if(NOT TARGET_PROCESSOR STREQUAL "aarch64")
+      list(APPEND source_files
+        darwin/secureboot.cpp
+      )
+    endif()
```

Comment:
I think doing this will be problematic -- I think it would be hard for a TLS server to differentiate configs based on archtecture. I'd be in favor of returning no rows on aarch64

Reply:
No rows, or a single row with empty fields?

Reply:
Depending on your agreed decision, feel free to roll back my last commit or ask me to.

Reply:
I don't think it's a _must have_. But I can tell you that as a TLS server, it's a bunch of weird additional work tracking the divergence across architectures. And it tends to create a lot of confusion. End users wonder why distributed queries don't work consistently, etc. 

I agree that as a project, we don't really have good answers for it.

Reply:
So I guess we can keep the table built, but I would not return an empty row, but no rows, as it is already for other tables when there's nothing to be found.

I will open a new draft blueprint to keep track of the desire to partially restore and properly implement the foreign tables concept.

Reply:
Behavior now changed to emit no results, and an INFO message, when the underlying functionality is unsupported (as in, on Apple Silicon). Test also updated to skip when running on macOS+ARM.